analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -284,8 +303,6 @@ private void writeGmeConfigMarkerFile() throws IOException {
                     }
 
                     RelaxedProjectVersionRef relaxedProjectVersionRef;
-                    // If we haven't found any original dependency we'll default to the current resolved dependency
-                    // value. This might be possible if the dependency has come from a lock file.
```

Comment:
I think this comment is still valid?

Reply:
Yeah, you are probably right. I removed when I was initially thinking of making some change here but ended up not doing some. I'll add it back

Reply:
Fixed